_posts/2024-03-25-whats-cooking.md

```diff
@@ -0,0 +1,224 @@
+
+# Used Technologies
+
+### Frontend
+
+- **Next.js:** A React framework that enables functionalities such as server-side rendering and generating static websites, enhancing SEO, performance, and user experience.
+
+- **Chakra UI:** A simple, modular and accessible component library that provides the building blocks to build React applications with speed and ease.
+
+### Backend
+
+- **Spring Boot:** Simplifies the development of new Spring applications through convention over configuration, offering a wide range of functionalities for modern web services.
+
+- **Spring AI:** Spring AI is a framework for building AI applications in Java, inspired by Spring principles for ease of development.
+
+- **LLM (Large Language Models):** Offers advanced capabilities in natural language processing, understanding, generation, and translation, enabling sophisticated interaction and content creation.
+
+### AWS Services
+
+- **ECR (Elastic Container Registry):** Provides a Docker container registry that makes it easy for developers to store, manage, and deploy Docker container images.
+
+- **ECS (Elastic Container Service):** A fully managed container orchestration service that makes it easy to run, stop, and manage containers on a cluster.
+
+- **App Runner:** Provides a managed service for developers to quickly deploy containerized web applications and APIs, without dealing with infrastructure management.
+
+- **Secret Manager:** Helps to protect access to applications, services, and IT resources without the upfront cost and complexity of managing a full infrastructure.
+
+- **DynamoDB:** A fast and flexible NoSQL database service for all applications that need consistent, single-digit millisecond latency at any scale.
+
+- **API Gateway:** Offers a fully managed service that makes it easy for developers to create, publish, maintain, monitor, and secure APIs at any scale.
+
+- **API Gateway API Key:** Utilized for controlling and monitoring API usage and access, enhancing security and usage metrics.
+
+### CI/CD Pipeline
+
+- **GitHub Workflows:** Automates your software development workflows.
+
+- **Terraform:** An infrastructure as code software tool that allows you to build, change, and version infrastructure safely and efficiently.
+
+### AI Models
+
+- **OpenAI Models:** Provides advanced AI models like GPT (Generative Pre-trained Transformer) for natural language understanding, generation, and conversational AI capabilities.
+
+- **AWS AI Model Bedrock Titan:** Bedrock Titan AI is a suite of large language models offered by Amazon for building generative AI applications.
+
+- **Cohere Command:** Cohere Command is a powerful language model designed for enterprises, excelling at following instructions and completing complex tasks.
+
+- **Meta Llama2:** Meta Llama2 is a next-generation large language model known for its commercial-grade capabilities and open-source availability.
```

Comment:
Verschillende beschrijving voor elk model hoeft niet echt. In se zijn dit allemaal language models. Enkel wanneer jullie extra features ervan (bv image generation) hebben gebruikt, zou ik dit nog expliciet vermelden.